app/services/create_batch.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBatch
+  def call
+    appointment_summaries = AppointmentSummary
+                            .joins('LEFT OUTER JOIN appointment_summaries_batches ' \
```

Comment:
I'll see what I can do, Arel has no concept of an outer join but there may be another way of finding `AppointmentSummary`s that don't have an associated `Batch`